Source/WebCore/crypto/mac/CryptoKeyECMac.cpp

```diff
@@ -254,49 +300,49 @@ RefPtr<CryptoKeyEC> CryptoKeyEC::platformImportSpki(CryptoAlgorithmIdentifier id
         return nullptr;
     index += bytesUsedToEncodedLength(keyData[index]) + 1; // Read length
     CCECCryptorRef ccPublicKey = nullptr;
-    if (doesUncompressedPointMatchNamedCurve(curve, keyData.size() - index)) {
-        if (CCECCryptorImportKey(kCCImportKeyBinary, keyData.data() + index, keyData.size() - index, ccECKeyPublic, &ccPublicKey))
+    if (!doesUncompressedPointMatchNamedCurve(curve, keyData.size() - index)) {
+        if (CCECCryptorImportKey(kCCImportKeyCompact, keyData.data() + index + 1, keyData.size() - index - 1, ccECKeyPublic, &ccPublicKey))
```

Comment:
These changes are probably unrelated.

Reply:
It is related since adding a new case there, needed such checks otherwise was throwing an error.